packages/fxa-auth-server/lib/luaScripts/setAccessToken_2.lua

```diff
@@ -1,11 +1,17 @@
 local userId = KEYS[1]
 local tokenId = KEYS[2]
 local token = ARGV[1]
-local ttl = ARGV[2]
+local limit = tonumber(ARGV[2])
+local ttl = tonumber(ARGV[3])
```

Comment:
Hmm, I made a comment on this but it got lost. 

If adding args, this filename should become `_3.lua`, ya?

Reply:
Narrator's voice: It did not get lost.